makefiles/boot/riotboot.mk

```diff
@@ -9,7 +9,10 @@ CFLAGS += -I$(BINDIR)/riotbuild
 
 HEADER_TOOL_DIR = $(RIOTBASE)/dist/tools/riotboot_gen_hdr
 HEADER_TOOL ?= $(HEADER_TOOL_DIR)/bin/genhdr
-BINDIR_APP = $(BINDIR)/$(APPLICATION)
+BINDIR_SUIT = $(BINDIR)/suit_files
```

Comment:
Actually bad name, no need to tie to suit here.

Reply:
What's a better name?

Reply:
Updated to a rioboot specific name, and used `BINDIR_SUIT` only for suit files (so manifests, not payloads)